js/templates/modals/listingDetail/listing.html

```diff
@@ -2,11 +2,11 @@
 
 <div class="topControls gutterHSm flex">
   <% if (ob.vendor) { %>
-  <div class="contentBox clrP clrBr clrT">
-    <div class="padSm gutterHSm overflowAuto flexVCent">
+  <div class="contentBox clrP clrSh3 clrBr clrT">
+    <div class="padSm gutterHSm overflowAuto margRSm flexVCent">
       <a class="clrBr2 clrSh1 disc storeOwnerAvatar flexNoShrink js-storeOwnerAvatar" style="<%= ob.getAvatarBgImage(ob.vendor.avatar) %>"></a>
-      <p class="txUnl clamp"><%= ob.vendor.name %></p>
-      <a class="link flexNoShrink js-goToStore"><% print(ob.openedFromStore ? ob.polyT('listingDetail.returnToStore'): ob.polyT('listingDetail.goToStore')) %></a>
+      <p class="txUnl tx3 clamp"><%= ob.vendor.name %></p>
+      <a class="link flexNoShrink tx6 margRSm js-goToStore"><% print(ob.openedFromStore ? ob.polyT('listingDetail.returnToStore'): ob.polyT('listingDetail.goToStore')) %></a>
```

Comment:
The margRSm here is overridden by the gutter class on the parent, so it can be removed. 

If it wasn't overridden, there would be double space to the right of the link, is that what was intended?